### DISCUSSION TITLE
OSPP 2023 Challenge - for #2437

### DISCUSSION THREAD

**Discussion by hydai** [WasmEdge/WasmEdge#2452] [Show and tell]
## Introduction

Dear Applicants,

We are excited to announce the pre-test for the OSPP project #2437, "Enabling WasmEdge Compilation with MSVC Toolchain". This project aims to evaluate your skills and qualifications for the program, and we encourage all interested candidates to participate.

## Requirements
1. Please complete the following tasks on a Microsoft Windows platform:
  A. Use the MSVC compiler to build WasmEdge and capture the error log.
  B. Use the Clang/GCC toolchain (the only successful way to build WasmEdge on Windows) to build WasmEdge, run the test suite, and capture the results. For the details, please refer to [our workflow files](https://github.com/WasmEdge/WasmEdge/blob/master/.github/workflows/reusable-build-on-windows.yml)

2. Once you have completed the tasks, please provide a link to your GitHub repository containing the following:
  A. A README file explaining how you approached the tasks and documenting your results.
  B. Screenshots or logs of the results from tasks 1A and 1B.

## Deadline

Please refer to [the OSPP 2023 timeline](https://summer-ospp.ac.cn/?lang=en) for the final deadline for valid submissions, which is June 4th, UTC+8.

We look forward to reviewing your submissions and thank you for your interest.

**Comment by am009** [comment]
Hi, here is my current progress:
https://github.com/am009/WasmEdge

**Reply by alabulei1** [reply]
Great work! Don't forget to submit your application via the OSPP platform.
